### PR TITLE
Snapshot UI Changes

### PR DESCRIPTION
**Pull request by Pezmc**
Fixes https://github.com/flowforge/flowforge/issues/1238

Screenshot 2022-11-23 at 17 25 41

### CODE REVIEW

File: frontend/src/pages/project/Snapshots/index.vue [hunk 1]
```diff
@@ -73,12 +63,30 @@ export default {
...
+                    s.deviceCount = deviceCounts[s.id]
+                    return s
+                })
+                this.loading = false
             }
-            this.loading = false
+        },
+        async countDevices () {
+            // hardcoded device limit to ensure all are returned - feels dirty
+            const data = await devicesApi.getDevices(null, 10000000)
```
Comment (Pezmc) [inline frontend/src/pages/project/Snapshots/index.vue:79]: I'm going to move this server-side as I agree it feels dirty and excessive just to count snapshots!

Comment (Pezmc) [reply frontend/src/pages/project/Snapshots/index.vue:79]: The devices API only works for **platform** admins. As such this needs to be **moved** server side.

File: package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
         "@fastify/csrf-protection": "^6.0.0",
         "@fastify/helmet": "^10.0.2",
         "@fastify/static": "^6.5.0",
-        "@flowforge/forge-ui-components": "^0.4.5",
+        "@flowforge/forge-ui-components": "^0.4.4",
```
Comment (joepavitt) [inline package.json:55]: reminder to bump this to 0.4.6

**Comment by Pezmc** [comment]
Screenshot 2022-11-23 at 17 25 41

**Review by joepavitt — APPROVED** [review]

**Pull request merged**